tests/fixture/cairo/runner-crate/src/lib.cairo

```diff
@@ -0,0 +1,34 @@
+// arrays2.cairo
+// Your task is to make the test pass without modifying the `create_array` function.
+// Make me compile and pass the test!
+// Execute `starklings hint arrays2` or use the `hint` watch subcommand for a hint.
+
+use array::ArrayTrait;
+use option::OptionTrait;
+
+// Don't modify this function
+fn create_array() -> Array<felt252> {
+    let mut a = ArrayTrait::new();
+    a.append(42);
+    a
+}
+
+fn remove_element_from_array(
+    ref a: Array<felt252>
+) { //TODO something to do here...Is there an array method I can use?
+    a.pop_front();
+}
+
+#[test]
```

Comment:
no `#[available_gas(20000000)]` ?

Reply:
This is copied over from another file for running. I'll gitignore this too...